### ISSUE TITLE
Automatic idle dimming

### ISSUE THREAD

**Issue by Bluscream** [xanderfrangos/twinkle-tray#363]
I wanted to disable the power plans "screen off" timer because it started giving me issues and use TwinkleTray to dim the screen down instead so i wrote an AHK script for it: https://github.com/Bluscream/ahk-scripts/blob/master/scripts/screendim.ahk

```
[01:10:48] is_idle: 0
[01:10:48] A_TimeIdlePhysical: 125
[01:10:48] idle_time_ms: 60000
[01:10:48] brightness_normal: 50
[01:10:48] brightness_idle: 0
[01:10:48] GetPowerProfileName(): Balanced
[01:10:48] ignored_power_profiles: ["High performance","Ultimate Performance","Bitsum Highest Performance","Driver Booster Power Plan"]
[01:10:48] A_Args: ["/menu"]
[01:10:48] twinkle_tray: {"directory":{"drive":"C:","path":"C://Users//blusc//AppData//Local//Programs//twinkle-tray//"},"drive":"C:","extension":"exe","fullname":"Twinkle Tray.exe","name":"Twinkle Tray","path":"C://Users//blusc//AppData//Local//Programs//twinkle-tray//Twinkle Tray.exe"}
```

**Comment by xanderfrangos** [comment]
HI @Bluscream,

Thanks for sharing! A similar feature is currently available in the beta version of Twinkle Tray: https://github.com/xanderfrangos/twinkle-tray/issues/296
 
It doesn't always behave well when resetting the brightness after the screens have turned off completely, but that shouldn't be an issue for you since you're using this instead of turning off the displays.

**Comment by Bluscream** [comment]
Dangit, i shouldve known that beforehand xD gonna use the beta instead (atleast i dont have to fix that myself if it breaks :D)

A thing i noticed is that even with brightness 0 the screen is still quite bright when looking at it at night. Could you incorporate a way to go below that by using some kind of overlay? Similar to what f.lux is doing but instead of overlaying a color temp filter just overlay a black image with different opacities?

**Comment by xanderfrangos** [comment]
Closing this issue, as the requested feature is available as of v1.14.0.

Regarding the f.lux functionality: applying an overlay isn't something I'm interested in doing at this time. But it's something I'll keep in mind for the future, as others have requested too.
